apps/reaper/test/integration/reaper/reaper_full_test.exs

```diff
@@ -335,6 +335,71 @@ defmodule Reaper.FullTest do
         60
       )
     end
+
+    @tag timeout: 120_000
```

Comment:
love to see this :(

Reply:
I think thats just copied from the other reaper full test, test